uwblueprint/jump-math
# feat: Add Classroom Modal

## Notion ticket link

[Add Classroom Modal](https://www.notion.so/uwblueprintexecs/Add-Classroom-Modal-d3d9a75835244834a335b656730c85db?pvs=4)


## Implementation description
* Created a new file `AddClassroomModal.tsx` which contains the add classroom modal
* Created a new file `SelectFormInputClassroom.tsx` which is the drop down select component 
* Created a new file `ClassroomTypes.tsx` which contain the fields for the form 
* Made a change to `TeacherPage.tsx` to incorporate the modal 


## Steps to test
1. Go to http://localhost:3000/teacher
2. Ensure the modal appears when clicking 'Add Classroom +' button on teacher page
3. Ensure the modal fields can be filled out 
4. Ensure the modal can be closed by clicking 'discard' and the user is back at the teacher page
5. Ensure the modal shows an error when fields are empty 
6. In the console, ensure that after the user hits 'submit' in the modal the field values are properly logged



## What should reviewers focus on?
* That the modal matches the design in figma


## Checklist
